### ISSUE TITLE
Dacryolith

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9940]
**Preferred term label:**
Dacryolith

**Synonyms**
Lacrimal calculus; Tear stone

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0155249

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Canaliculitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

term added as HP:6000131
